.claude/commands/tasks.md

```
@@ -0,0 +1,26 @@
+Read the task store at `.reagent/tasks.jsonl` and render a clean summary table.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)